### PR TITLE
Adding more countries and Aligning the items

### PR DESCRIPTION
**Pull request by IndrajaChelimela**
Added 2 more countries, canada and malaysia to toolbar and drawer. And aligned the items. 
Changing the countries in toolbar and drawer are not working together and ill update this in next pull request

### CODE REVIEW

File: src/components/howDoWeOperate/LayoutVideoCard.vue [hunk 1]
```diff
@@ -97,6 +97,16 @@ export default Vue.extend({
         fCclass: "ma-2 white--text font-weight-bold red lighten-1",
         bCclass: "ma-2 white--text font-weight-bold red lighten-1",
         sMclass: "ma-2 red--text font-weight-bold white darken-1"
+      },
+      CANADA: {
+         fCclass: "ma-2 white--text font-weight-bold red lighten-1",
```
Comment (haigopi) [inline src/components/howDoWeOperate/LayoutVideoCard.vue:102]: Please format the code properly before commit.

File: src/components/howDoWeOperate/LayoutSignInBeAChefCard.vue [hunk 2]
```diff
@@ -17,24 +17,29 @@
...
+      <v-spacer />
     <v-card-actions :class="this.$vuetify.breakpoint.smAndDown ? 'd-flex flex-column ma-5' : ''">
       <div class="text-center">
         <v-btn class="white--text font-weight-bold red darken-1 ">
           <v-icon left>mdi-chef-hat</v-icon>
           Reserve
         </v-btn>
-
-        <v-btn class="white--text font-weight-bold blue darken-1 ml-5"> Existing Chef? <v-icon class="ml-8 mr-1">mdi-key</v-icon> Login </v-btn>
+        <v-btn class="white--text font-weight-bold blue darken-1 ml-4"> Existing Chef? <v-icon >mdi-key</v-icon> Login </v-btn>
```
Comment (haigopi) [inline src/components/howDoWeOperate/LayoutSignInBeAChefCard.vue:30]: Add some screenshots as well because of this code chnage. Please.

Comment (IndrajaChelimela) [reply src/components/howDoWeOperate/LayoutSignInBeAChefCard.vue:30]: After decreasing the width of textbox and moving buttons, those two items are not aligned
Screenshot (9)
So, i decreased the height of the image and increased the passing of textbox. It looks like this
Screenshot (10)

**Comment by haigopi** [comment]
I am approving this for now. Please fix the rest of the stuff as discussed in the last session and create a new PR as incremental delivery

**Pull request merged**